clients/src/main/resources/common/message/ElectLeadersRequest.json

```diff
@@ -32,7 +32,9 @@
         { "name": "Topic", "type": "string", "versions": "0+", "entityType": "topicName", "mapKey": true,
           "about": "The name of a topic." },
         { "name": "Partitions", "type": "[]int32", "versions": "0+",
-          "about": "The partitions of this topic whose leader should be elected." }
+          "about": "The partitions of this topic whose leader should be elected." },
+        { "name": "DesiredLeaders", "type": "[]int32", "versions": "3+", "nullableVersions": "3+",
+          "about": "The desired leaders. The entry should match with the entry in Partitions by the index." }
```

Comment:
Please specify that this is only for DESIGNATED elections. Also, please change the name to `designatedLeaders` (we shouldn't mix terminiology -- pick one term, like "designated", and stick with it, rather than having designated / desired / specified all over the place.)